src/player.h

```diff
@@ -594,6 +594,7 @@ class Player final : public Creature, public Cylinder
 		bool isImmune(ConditionType_t type) const final;
 		bool hasShield() const;
 		bool isAttackable() const final;
+		bool lastHitIsPlayer(Creature* lastHitCreature) const;
```

Comment:
I think he means moving the function into player.cpp and not declaring it in the header file. You could also just make it static.